website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -282,17 +265,10 @@ There are three ways to run the operator:
 - As a Deployment inside a Kubernetes cluster
 - Managed by the [Operator Lifecycle Manager (OLM)][doc-olm] in [bundle][quickstart-bundle] format
 
-### 1. Run locally outside the cluster
 
-Execute the following command, which install your CRDs and run the manager locally:
+The following steps will show how to deploy the operator on the Cluster. However, to run locally for development purposes and outside of a Cluster use the target `make install run`.
 
-```sh
-make install run ENABLE_WEBHOOKS=false
-```
-
-### 2. Run as a Deployment inside the cluster
```

Comment:
Are you removing these numbered steps so the tutorial only describes one method of deployment?

Reply:
IMO was easier to understand without the number list. However, based on your previuous comment. and this one I reverted it.